examples/cp-all-in-one/docker-compose.yml

```diff
@@ -122,3 +122,18 @@ services:
     environment:
       KSQL_CONFIG_DIR: "/etc/ksql"
       KSQL_LOG4J_OPTS: "-Dlog4j.configuration=file:/etc/ksql/log4j-rolling.properties"
+
+  rest-proxy:
+    image: confluentinc/cp-kafka-rest:4.1.0
```

Comment:
4.1.1 is the latest version

Reply:
The version issue affects all other products too. Not sure why the version wasn't updated when 4.1.1 was release.

Reply:
Is it not the case that the `4.1.0-post` branch should use docker image 4.1.0, and the `4.1.1-post` branch should use docker image 4.1.1, etc?  (this question applies to all container images, not just the rest proxy).  And if yes, how does the image version get properly propagated branch to branch?  @aayars